metricbeat/module/apache/status/_meta/testdata/docs.plain-expected.json

```diff
@@ -19,7 +19,7 @@
                     "system": 0.2,
                     "user": 0.16
                 },
-                "hostname": "127.0.0.1:43985",
+                "hostname": "127.0.0.1:34545",
```

Comment:
This is the only thing that changed after regenerating all files.

Reply:
I'm surprised this changed (it shouldn't). @sayden I remember we had some discussion around apache and "whitelist" some files which should not be remove or similar?

Reply:
Yes, I finally added a config in `TestAll` to omit specific fields or fields group from the comparison because they were changing in each regenerate https://github.com/elastic/beats/pull/11714/files#diff-5b51ecdcb61bd7977eb4d2e2852178c0R32

Reply:
And why did it still change here?

Reply:
@sayden ^